components/unzipped/dashboard/project/templates.js

```diff
@@ -0,0 +1,561 @@
+                        </Button>
+                    </Absolute>
+                </Modal>
+            )}
+            {createTask && (
+                <Modal onHide={() => setCreateTask(false)} height="550px" background="#D9D9D9">
+                    <FormField
+                        fieldType="input"
+                        margin
+                        fontSize='14px'
+                        noMargin
+                        width="95%"
+                        onChange={(e) => {
+                            setTaskForm((prev) => ({
+                                ...prev,
+                                taskName: e.target.value
+                            }));
+                        }}
+                        handleEnterKey={() => { }}
+                        value={taskForm?.taskName}
+                    >
+                        TASK NAME(REQUIRED)
+                    </FormField>
+                    <Grid3 margin="0px" width="95%" grid="2fr 2fr">
+                        <FormField
+                            fieldType="input"
+                            margin
+                            fontSize='14px'
+                            noMargin
+                            width="95%"
+                            onChange={(e) => {
+                                e.persist();
+                                const inputValue = e.target.value;
+                                const regex = /^([0-9]+)?$/;
+                                if (regex.test(inputValue)) {
+                                    setTaskForm((prev) => ({
+                                        ...prev,
+                                        hours: +inputValue
+                                    }));
+                                }
+                            }}
+                            handleEnterKey={() => { }}
+                            value={taskForm?.hours}
+                        >
+                            HOURS
+                        </FormField>
+                        <FormField
+                            fieldType="input"
+                            margin
+                            fontSize='14px'
+                            noMargin
+                            width="100%"
+                            onChange={(e) => {
+                                e.persist();
+                                const inputValue = e.target.value;
+                                const regex = /^([0-9]+)?$/;
+                                if (regex.test(inputValue)) {
+                                    if (inputValue < 9) {
+                                        setTaskForm((prev) => ({
+                                            ...prev,
+                                            storyPoints: +inputValue
+                                        }))
+                                    }
+                                }
+                            }}
+                            handleEnterKey={() => { }}
```

Comment:
do we need to handle anything here?

Reply:
No, but it is required. Otherwise it throws the error.